datasync/chaindatafetcher/kas/contract_caller_test.go

```diff
@@ -156,6 +157,19 @@ func (s *SuiteContractCaller) TestContractCaller_SupportsInterface_NoContractCod
 	s.Nil(err)
 }
 
+func (s *SuiteContractCaller) TestContractCaller_SupportsInterface_ErrVMDefault() {
```

Comment:
I think we could integrate `TestContractCaller_SupportsInterface_ErrVMDefault`, `TestContractCaller_SupportsInterface_NoContractCode`, `TestContractCaller_SupportsInterface_ExecutionReverted`, and `TestContractCaller_SupportsInterface_EmptyResult` into one test case.

Reply:
As discussed in f2f, I opened another PR.